tools/mongo_flag.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# This script allow to add or remove a flag for a user
+
+LOGIN=$1
+# is_internal|is_fake|never_expire
+FLAG=$2
+# true|false
+BOOL=$3
+
+mongo gomngr --quiet --eval 'db.users.find({"uid" : "'${LOGIN}'"}).forEach(
```

Comment:
should be findOne if expecting to match 1 user (don't know impact of foreach in this case...)

Reply:
Just launch it here at Roscoff :)
Seems ok but yes, I will change that

Reply:
I found [`findOneAndUpdate`](https://www.mongodb.com/docs/manual/reference/method/db.collection.findOneAndUpdate/).